src/USER-MISC/fix_pafi.cpp

```diff
@@ -19,6 +19,28 @@
 
 
 #include "fix_pafi.h"
+#include "mpi.h"
+#include "math.h"
```

Comment:
this is getting worse, not better. please read: https://github.com/lammps/lammps/blob/master/doc/include-file-conventions.md

Reply:
apologies!